command/ca/certificate.go

```diff
@@ -259,10 +328,21 @@ func (f *certificateFlow) getClient(ctx *cli.Context, subject, tok string) (caCl
 	return ca.NewClient(caURL, options...)
 }
 
+type errACMEURL struct {
+	id string
```

Comment:
It doesn't look we need this id. By my other comments, we might not need this type.

Reply:
removed